c2corg_api/views/area.py

```diff
@@ -47,13 +47,13 @@ def put(self):
         return self._put(Area, schema_area, after_update=update_associations)
 
 
-def insert_associations(area):
+def insert_associations(area, user_id):
```

Comment:
OK, so you mean that ``user_id`` is passed anyway when calling the ``after_add`` hook and that the hooked methods should have it in their signature, even though not actually using it?